src/main/kotlin/com/terraformation/backend/seedbank/model/AccessionHistoryModel.kt

```diff
@@ -41,6 +42,11 @@ data class AccessionHistoryModel(
     val type: AccessionHistoryType,
     /** If the event was attributed to a Terraware user, that user's ID. */
     val userId: UserId?,
+    /**
+     * The associated batch ID, if applicable. Batches that are created during nursery transfers are
+     * associated to the history entry.
+     */
+    val batchId: BatchId? = null,
```

Comment:
Move this up so the fields stay alphabetical.

Reply:
Should this still be moved up, even though it is an optional parameter?

Reply:
Never mind, I see we can use named arguments